app/src/main/res/layout/fragment_login.xml

```diff
@@ -1,25 +1,116 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Scroll view can be the root of this layout.
You remove those 2 imageViews above, export the background itself as an svg from Figma and set the it as background drawablet to the inner LinearLayout